packages/web-react/src/components/Stack/README.md

```diff
@@ -32,12 +32,12 @@ Advanced example usage:
 
 | Prop name                 | Type     | Default | Required | Description                            |
 | ------------------------- | -------- | ------- | -------- | -------------------------------------- |
-| `class`                   | `string` | `null`  | no       | Custom CSS class                       |
 | `elementType`             | `string` | `div`   | no       | Element type of the wrapper element    |
 | `hasEndDivider`           | `bool`   | `false` | no       | Render a divider after the last item   |
 | `hasIntermediateDividers` | `bool`   | `false` | no       | Render dividers between items          |
 | `hasSpacing`              | `bool`   | `false` | no       | Apply a spacing between items          |
 | `hasStartDivider`         | `bool`   | `false` | no       | Render a divider before the first item |
+| `UNSAFE_className`        | `string` | `null`  | no       | Wrapper custom classname               |
```

Comment:
Shouldn't be escape hatches mentioned below the table as in [Twig package](https://github.com/lmc-eu/spirit-design-system/blob/59c4fbf8d7ce658bcab7dae7d5f5d9f55f498956/packages/web-twig/src/Resources/components/Stack/README.md?plain=1#L62)?

cc @literat

Reply:
This problem is in all readmes. I have file an issue for this.